### PR TITLE
CB-7984 Proxies,Tags,Telemetry should be PowerUser Only APIs

### PR DESCRIPTION
**Pull request by sodre90**

### CODE REVIEW

File: environment/src/main/java/com/sequenceiq/environment/proxy/v1/controller/ProxyController.java [hunk 1]
```diff
@@ -55,39 +55,45 @@ public ProxyController(ProxyConfigService proxyConfigService,
...
         String accountId = ThreadBasedUserCrnProvider.getAccountId();
         ProxyConfig config = proxyConfigService.getByNameForAccountId(name, accountId);
         return proxyConfigToProxyResponseConverter.convert(config);
     }
 
     @Override
-    @CheckPermissionByAccount(action = AuthorizationResourceAction.ENVIRONMENT_READ)
-    public ProxyResponse getByEnvironmentCrn(@ResourceCrn @TenantAwareParam String environmentCrn) {
+    @InternalOnly
+    public String getCrnByAccountIdAndName(String accountId, String name) {
```
Comment (horadla23) [inline environment/src/main/java/com/sequenceiq/environment/proxy/v1/controller/ProxyController.java:75]: `accountId` should be annotated with `@AccountId`, if the method is internal only

Comment (sodre90) [reply environment/src/main/java/com/sequenceiq/environment/proxy/v1/controller/ProxyController.java]: Thanks!

**Comment by oleewere** [comment]
are you sure about this? like for account telemetry, you won't be able to see details about Telemetry (i mean because of the restricted read operations, i guess putting that to writes are fine)

**Comment by horadla23** [comment]
> are you sure about this? like for account telemetry, you won't be able to see details about Telemetry (i mean because of the restricted read operations, i guess putting that to writes are fine)

yes, this is based on authz sharing design agreement

**Pull request merged**